specification/security/resource-manager/readme.md

```diff
@@ -175,10 +175,10 @@ input-file:
 - Microsoft.Security/stable/2019-08-01/deviceSecurityGroups.json
 - Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json
 - Microsoft.Security/stable/2019-08-01/iotSecuritySolutionAnalytics.json
-- Microsoft.Security/stable/2019-08-01/iotAlertTypes.json
-- Microsoft.Security/stable/2019-08-01/iotAlerts.json
-- Microsoft.Security/stable/2019-08-01/iotRecommendationTypes.json
-- Microsoft.Security/stable/2019-08-01/iotRecommendations.json
+#- Microsoft.Security/stable/2019-08-01/iotAlertTypes.json
```

Comment:
Why add '#' here

Reply:
I didn't want to entirely delete it because it belongs to a completely different team, so I didn't want to make a terminal change as deleting their old versions..